model/host/hostutil_test.go

```diff
@@ -512,29 +512,32 @@ func TestJasperClient(t *testing.T) {
 	}
 }
 
+// withJasperServiceSetupAndTeardown performs necessary setup to start a
+// Jasper RPC service, executes the given test function, and cleans up.
+func withJasperServiceSetupAndTeardown(ctx context.Context, env *mock.Environment, manager *jasper.MockManager, h *Host, fn func()) error {
+	catcher := grip.NewBasicCatcher()
+	if !catcher.HasErrors() {
```

Comment:
This will never be false.

I think it might be more clear to write this function without a catcher.